app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/viewmodel/MovieListViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.marcinwisniewski.intivetrainingapp.movielist.viewmodel
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.Movie
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieRepository
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
+import javax.inject.Inject
+
+class MovieListViewModel @Inject constructor(private val movieRepository: MovieRepository) : ViewModel() {
+
+    private val mutableMovieList: MutableLiveData<List<Movie>> = MutableLiveData()
+    val movieList: LiveData<List<Movie>> = mutableMovieList
+    private lateinit var movieListDisposable: Disposable
```

Comment:
i was not sure if it should be a single disposable or compositedisposable

Reply:
CompositeDisposable is for more subscriptions